src/model/exceptions/last-part-in-rundown-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { Exception } from './exception'
+import { ErrorCode } from '../enums/error-code'
+
+export class LastPartInRundownException extends Exception {
```

Comment:
Could it make sense to merge this and `LastPartInSegmentExcepion`, and then in the exception message include the detail about what entity it was the last part of?

Reply:
Potentially yes.
But this is used for more than just writing to the log what it is the end of.
There is different business logic that needs to be executed depending on if it's the last Part of the Segment or the last Part of the Rundown. So you would still need to differentiate between the now "subTypes" of the exception that this suggestion will introduce, unless you want to rely on a magic string.

Since there is different business logic dependent on the different scenarios, I would argue it make sense to keep the two exceptions separate.

Reply:
That fair, didn't think about the fact that there might be other than logging being run depending on the exception thrown.